PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties

```diff
@@ -18,10 +18,10 @@ applicationinsights.key=57cc111a-36a8-44b3-b044-25d293b8b77c
 updateVersionRange=false
 patchPluginXmlSinceBuild=222
 
-kotlin_version=1.6.10
+kotlin_version=1.6.20
 # note: jackson_kotlin_version also needs updating in Utils\azuretools-core\pom.xml
 jackson_kotlin_version=2.13.2
-gradle_plugin_version=1.6.0
+gradle_plugin_version=1.7.0-SNAPSHOT
```

Comment:
Temporary, I hope 😅

Reply:
Reverting it here :) https://github.com/JetBrains/azure-tools-for-intellij/pull/608